web/modules/custom/do_base/do_base.deploy.php

```
@@ -1318,6 +1318,57 @@ function do_base_deploy_add_related_posts_block(): string {
+ *
+ * The component is placed without images so an author can fill it from the
+ * media library. It renders nothing until they do, so an empty band never
+ * reaches a visitor.
+ */
+function do_base_deploy_add_homepage_image_list(): string {
+  $node = _do_base_front_page_node();
+
+  if (!$node instanceof NodeInterface) {
+    Helper::reporter()->skipped('The front page is not a node, so there is nothing to place the image list on.');
+
+    return Helper::report();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\drupal_helpers\Helper' in method 'do_base_deploy_add_homepage_image_list'. (undefined)

(StaticAccess)